data/default_release_images.json

```diff
@@ -85,5 +85,17 @@
         "url": "quay.io/openshift-release-dev/ocp-release:4.12.0-rc.4-aarch64",
         "support_level": "beta",
         "version": "4.12.0-rc.4"
-    }
-]
\ No newline at end of file
+    },
+    {
+      "openshift_version": "4.12.0-rc.4-multi",
+      "cpu_architecture": "multi",
+      "cpu_architectures": [
+          "x86_64",
+          "arm64",
+          "ppc64le",
+          "s390x"
+      ],
+      "url": "quay.io/openshift-release-dev/ocp-release:4.12.0-rc.4-multi",
+      "version": "4.12.0-rc.4-multi"
+  }
```

Comment:
indentation is weird here. can you fix it?

Reply:
Yes you are right